DMTN-107.tex

```diff
@@ -67,14 +68,18 @@ \subsection{Alert Production}\label{ssec:req_ap}
 
 \subsection{Alert Packet Contents}\label{ssec:req_apc}
 
-The Data Products Definition Document \citeds{LSE-163} specifies that the alert packets include {\em at least} a 12 month history, and cutouts of the template and difference images (Section 3.5.1).
-The 12 month history is not a true requirement, and should be interpreted as applying only after 12 months of LSST operations.
+\citeds{LSE-61} contains a formal requirement that the alert packets include {\em at least} a 12 month history (if available), as well as cutouts of the template and difference images\dmreq{0274}.
```

Comment:
It would be reasonable to still mention that DPDD 3.5.1 confirms/echos this requirement.

Reply:
I should never pass up a chance to direct readers to the DPDD! Added a sentence: "Alert packet contents are also described in Section 3.5.1 of \citeds{LSE-163}."

Reply:
Maybe "The availability of the 12 month history and the cutouts are also discussed in Section 3.5.1 of \citeds{LSE-163}."?  Just to keep it very concrete?

Reply:
yeah! 👍 that sounds better, i'll use that